robot.py

```diff
@@ -79,13 +89,22 @@ def teleopPeriodic(self):
         ))
 
         # Tests the button debouncer
-        self.sd.putNumber((int) self.button.get())
+        self.sd.putNumber(self.button.get())
         
         # Tests ColorSensor output
         self.sd.putNumber(self.colorSensor.getGreen())
 
         # Tests the LimeLight's latency
         self.sd.putNumber(self.limelight.getLatency())
```

Comment:
`self.sd.putNumber()` requires a label before the value. i.e. `self.sd.putNumber("Button", self.button.get())`

Reply:
Fixed on master